README.md

```diff
@@ -27,20 +27,93 @@ The `zone` key is purely static and arbitrary, but by having different `zone` va
 
 ### Run the application
 
-1.  Run `redis` on the default port `6379` and set a default `zone` name:
+1.  Run `redis` 
+
+    - (Kubernetes setup) on the default port `6379`  and set a default `zone` name:
 
     ```sh
     $ redis-server
     $ redis-cli set zone local
     ```
 
-1.  Start `demo-app` on the default port `5000`:
+    - (Universal setup) on port `26379` and set a default `zone` name:
+
+     ```sh
+    $ redis-server --port 26379
+    $ redis-cli set zone local
+    ```
+
+2.  Start `demo-app` on the default port `5000`:
 
     ```sh
     $ npm start --prefix=app/
     ```
 
-1.  Navigate to [`127.0.0.1:5000`](http://127.0.0.1:5000) and increment the counter!
+3.  Navigate to [`127.0.0.1:5000`](http://127.0.0.1:5000) and increment the counter!
```

Comment:
It might be worth saying that if the DPs are not started it won't work?

Reply:
I think we might think about removing this step? Or add info that it only works in Kubernetes steps.